src/ripple/nodestore/ShardSizeTuning.md

```diff
@@ -0,0 +1,83 @@
+# Shard Size Tuning
+
+The purpose of this document is to compare shard size of 8K and 16K ledgers.
+
+## Methodology
+
+One can see visually from a block explorer that typical mainnet ledger
+consists of about 30 offer transactions issued by about 8 different accounts,
+and several transactions of other types. To simulate this situation and
+similar situations we had constructed deterministic shards of differenet sizes,
+with different number of offer transactions per ledger and different number
+of accounts issuing these offers.
+
+Number of ledgers per shard starts from 256 to 16K with doubling the size on
+each step. We considered the following numbers of offers per ledger:
+0, 1, 5, 10 and 30. Also we considered case of 1 and 8 accounts issuing offers.
+For each constructed deterministic shard we counted its size. Finally we
```

Comment:
At this point in the document it isn't clear why you're comparing the doubled size of a shard containing _n_ ledgers with the size of a shard containing _2n_ ledgers. Please state why this comparison is necessary.

Reply:
Done.